Neos.Flow/Classes/Security/Authentication/Provider/AbstractProvider.php

```diff
@@ -30,6 +29,16 @@
      */
     protected $options = [];
 
+    /**
+     * @param string $name
+     * @param array $options
+     * @return AuthenticationProviderInterface
+     */
+    public static function create(string $name, array $options)
+    {
+        return new static($name, $options);
```

Comment:
Could we now even make the constructor protected, so we don't have to care about unintentional instanciation?